src/subcommands/generate.rs

```diff
@@ -1055,6 +1103,12 @@ fn resolve_types(
                     serde_as_array: true,
                     extra_ref_type: true,
                     fields: request_fields,
+                    derives: additional_derives_types
+                        .additional_derives_types
+                        .iter()
+                        .find(|typ| typ.name == rusty_name)
+                        .map(|t| t.derives.clone())
+                        .unwrap_or_default(),
```

Comment:
Much duplication of the same snippet. Better implement it as a method for `AdditionalDerivesOptions`.